src/ec/packages/ec-component-expandable/expandable.html.twig

```diff
@@ -83,7 +75,7 @@
   <div
     id="{{ _id ~ '-content' }}"
     class="ecl-expandable__content"
-    {{ _content_hidden_attribute }}
+    {% if not _expanded %}hidden{% endif %}
```

Comment:
Same as before, this is handled by javascript, we set it as hidden and then js will play with that. I would have honestly considered this as an improvement in case it was meant to provide a non javascript solution with the collapsed button hidden and the content revealed, but it's not the case, so to stick to the definition of the component as we have it ECL we set this as hidden by default and then we leave the task to the js.

Reply:
Same as my previous comment this was already implemented... See: https://github.com/ec-europa/ecl-twig/blob/292c833c54b31c19edfdd97042fb16dc5cb405a5/src/ec/packages/ec-component-expandable/expandable.html.twig#L66

Updated now.